gcp/appengine/frontend_handlers.py

```diff
@@ -190,6 +191,28 @@ def add_source_info(bug, response):
   response['source_link'] = response['source']
 
 
+def add_related_aliases(bug: osv.Bug, response):
+  """Add links to other osv entries that's related through aliases"""
+  # Add links to other entires if they exist
+  aliases = []
+  for alias in bug.aliases:
+    result = bug.get_by_id(alias)
+    if result:
+      aliases.append((alias, True))
```

Comment:
Can we put this in dicts instead? So that the template can refer to fields like:

`alias.id` and `alias.exists`